ledger-tool/src/output.rs

```diff
@@ -0,0 +1,51 @@
+use {
+    serde::Serialize,
+    solana_cli_output::{QuietDisplay, VerboseDisplay},
+    std::fmt::{Display, Formatter, Result},
+};
+
+#[derive(Serialize, Debug, Default)]
+pub struct SlotInfo {
+    pub start: u64,
+    pub end: u64,
+    pub total: usize,
+    #[serde(rename = "fromLast", skip_serializing_if = "Option::is_none")]
+    pub from_last: Option<u64>,
+}
+
+impl Display for SlotInfo {
```

Comment:
I'm not very clear on what these Display implementations are for, since we're not using them in `main.rs`. Am I missing something?

Reply:
No it's related to previous implementation based on `bounds.rs` module, following `bigtable.rs` subcommand as reference; i tought it was the idea to refactor `ledger-tool` crate, so i followed it :)
Considering what you suggested after reviewing sources, i guess that implementing the `Display` trait is ok but its logic should be the one used for "simple text printing". Did i understand correctly? Thank you :)

Reply:
> i guess that implementing the Display trait is ok but its logic should be the one used for "simple text printing"

Yes, implementing the Display trait is fine; however, CriesofCarrots comment is pointing out that as you have it, you are not using utilizing the Display trait implementation. Because we are implementing Display trait, this
```rust
                        match output_format {
                            OutputFormat::Json | OutputFormat::JsonCompact => {
                                println!("{}", output_format.formatted_string(&slot_bounds))
                            }
                            _ => {
                                // Simple text print in other cases.
                                ...
```
should just become
```rust
                        println!("{}", output_format.formatted_string(&slot_bounds))
```

Reply:
Got it, i'll update as suggested so. Thanks!